Source/FixedBackground/FixedBGSimulationParametersBase.hpp

```diff
@@ -0,0 +1,153 @@
+/* GRChombo
+ * Copyright 2012 The GRChombo collaboration.
+ * Please refer to LICENSE in GRChombo's root directory.
+ */
+
+#ifndef FIXEDBGSIMULATIONPARAMETERSBASE_HPP_
+#define FIXEDBGSIMULATIONPARAMETERSBASE_HPP_
+
+// General includes
+#include "BoundaryConditions.hpp"
+#include "ChomboParameters.hpp"
+#include "GRParmParse.hpp"
+#include "SphericalExtraction.hpp"
+
+// add this type alias here for backwards compatibility
+using extraction_params_t = SphericalExtraction::params_t;
+
+class FixedBGSimulationParametersBase : public ChomboParameters
```

Comment:
Check that this has the same formatting etc as in the more recent updates to SimulationParametersBase.hpp